### PR TITLE
Fix appearance of search text and hide auto-changes to lowercase from user.

### PR DESCRIPTION
**Pull request by RonakLakhotia**
Fix [#394](https://github.com/reposense/RepoSense/issues/394)

### CODE REVIEW

File: frontend/src/static/js/v_summary.js [hunk 1]
```diff
@@ -168,7 +168,8 @@ window.vSummary = {
     getFilterHash() {
       const { addHash } = window;
 
-      this.filterSearch = this.filterSearch.toLowerCase();
+
+      this.filterSearch = this.filterSearch;
```
Comment (yamidark) [inline frontend/src/static/js/v_summary.js]: This line doesn't do anything anymore right?

File: frontend/src/static/js/v_summary.js [hunk 2]
```diff
@@ -256,12 +257,12 @@ window.vSummary = {
...
         // filtering
         repo.users.forEach((user) => {
-          const toDisplay = this.filterSearch
+          const toDisplay = this.filterSearch.toLowerCase()
             .split(' ').filter(param => param)
             .map(param => user.searchPath.search(param) > -1)
             .reduce((curr, bool) => curr || bool, false);
 
-          if (!this.filterSearch || toDisplay) {
+          if (!this.filterSearch.toLowerCase() || toDisplay) {
```
Comment (yamidark) [inline frontend/src/static/js/v_summary.js]: No need to convert lower case here, rather if `this.filterSearch` is `null` then we would get an error instead.

File: frontend/src/static/js/v_summary.js (hunk 1 above)
Comment (RonakLakhotia) [reply frontend/src/static/js/v_summary.js]: Yes it is a redundant line, will remove it.

File: frontend/src/static/js/v_summary.js (hunk 2 above)
Comment (RonakLakhotia) [reply frontend/src/static/js/v_summary.js]: Oh right, makes sense. 👍

Comment (RonakLakhotia) [reply frontend/src/static/js/v_summary.js]: @yamidark requesting review

**Pull request merged**